mlflow_torchserve/__init__.py

```diff
@@ -1,26 +1,40 @@
-import docker
 import json
-import subprocess
-import time
 import logging
 import os
-import pandas as pd
-import torch
+import subprocess
+import time
 from pathlib import Path
 
+import docker
+import pandas as pd
 import requests
-from mlflow_torchserve.config import Config
-
+import torch
 from mlflow.deployments import BaseDeploymentClient, get_deploy_client
-from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+from mlflow.deployments import PredictionsResponse
 from mlflow.models.model import Model
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+
+from mlflow_torchserve.config import Config
 
 _logger = logging.getLogger(__name__)
 
 _DEFAULT_TORCHSERVE_LOCAL_INFERENCE_PORT = "8080"
 _DEFAULT_TORCHSERVE_LOCAL_MANAGEMENT_PORT = "8081"
 
 
+class CustomPredictionsResponse(PredictionsResponse):
```

Comment:
Nit: Do we want to call this `TorchServePredictionsResponse`?

Reply:
@dbczumar I have done this change as a separate [PR](https://github.com/mlflow/mlflow-torchserve/pull/98). Can you please review it when you find time. The builds are passing.